crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po

```diff
@@ -1182,9 +1182,11 @@ msgstr ""
 msgid "Any supporting materials (please list and describe them)."
 msgstr "Cualquier material de apoyo (haga una lista y describa cada artículo)."
 
-#: templates/base.html:22 templates/base.html:40
-msgid "Civil Rights Division - Department of Justice"
-msgstr "La División de Derechos Civiles - Departamento de Justicia"
+#: templates/base.html:22 templates/base.html:40 templates/forms/base.html:19
+#: templates/forms/report_base.html:13
+msgid "Contact the Civil Rights Division | Department of Justice"
+msgstr ""
+"Comuníquese con la División de Derechos Civiles - Departamento de Justicia"
```

Comment:
We want to change the `-` here to `|` too (spanish version)

Reply:
@harikrishnagit ^

Reply:
Sorry.. I keep missing one everytime. I think this is complete now. please review